crates/hc_demo_cli/Cargo.toml

```diff
@@ -20,22 +20,18 @@ hdk = { path = "../hdk", version = "^0.6.0-dev.6", optional = true }
 holochain = { path = "../holochain", version = "^0.6.0-dev.9", optional = true, default-features = false }
 holochain_types = { path = "../holochain_types", version = "^0.6.0-dev.9", optional = true }
 holochain_keystore = { path = "../holochain_keystore", version = "^0.6.0-dev.6", optional = true }
-rand = { version = "0.8.5", optional = true }
-rand-utf8 = { version = "0.0.1", optional = true }
-rustls = "0.23.25"
+rand = { version = "0.9", optional = true }
+rand-utf8 = { version = "0.0.2", optional = true }
+rustls = "0.23.28"
 serde = { version = "1", optional = true }
 tokio = { version = "1.27", features = ["full"], optional = true }
 tracing = { version = "0.1.37", optional = true }
 tracing-subscriber = { version = "0.3.16", optional = true }
+holochain_serialized_bytes = "=0.0.56"
 
 [dev-dependencies]
 tempfile = "3.5.0"
 
-# When building for the WASM target, we need to configure getrandom
-# to use the host system for the source of crypto-secure randomness.
-[target.'cfg(all(target_arch = "wasm32", target_os = "unknown"))'.dependencies]
-getrandom = { version = "0.2", features = ["custom"] }
```

Comment:
Custom feature is gone! https://github.com/rust-random/getrandom/pull/504

Reply:
Yay